src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+document.addEventListener('click', function (e) {
+  const link = e.target.closest('a');
+
+  if (!link) {
+    return;
+  }
+
+  e.preventDefault();
+
+  const largeImageUrl = link.getAttribute('href');
```

Comment:
Ensure that the `href` attribute contains a valid URL for the image. You might want to add error handling in case the URL is invalid or the image fails to load.